src/components/01-branding/02-typography/fields/fields.njk

```diff
@@ -11,57 +11,57 @@
     <tr>
       <td><element class="sds-field"><h1>Field Header</h1></element></td>
       <th scope="row">&ltelement class="sds-field">&lth1>&lt/h1>&lt/element></th>
-      <td>Weight: Semi-Bold / Size: 1.46rem / Line Height: 1.1rem / Color: Base-Dark (#454540)</td>
+      <td>Weight: Semi-Bold / Size: 1.46rem / Line Height: 1.1 / Color: Base-Dark (#454540)</td>
     </tr>
     <tr>
       <td><element class="sds-field"><h1 class="sds-light">Field Header Light</h1></element></td>
       <th scope="row">&ltelement class="sds-field">&lth1 class="sds-light">&lt/h1>&lt/element></th>
-      <td>Weight: Light / Size: 1.13rem / Line Height: 1.1rem / Color: Base-Dark (#454540)</td>
-    </tr>       
+      <td>Weight: Semi-Bold / Size: 1.13rem / Line Height: 1.1 / Color: Base-Dark (#454540)</td>
+    </tr>
     <tr>
       <td><element class="sds-field">Field Label</element></td>
       <th scope="row">&ltelement class="sds-field">&lt/element></th>
-      <td>Weight: Regular / Size: 0.93rem / Line Height: 1.5rem / Color: Base-Dark (#454540)</td>
+      <td>Weight: Regular / Size: 0.93rem / Line Height: 1.3 / Color: Base-Dark (#454540)</td>
     </tr>
     <tr>
       <td><element class="sds-field--large">Large Field Label</element></td>
```

Comment:
There is this non-standard element tag throughout, please use standard HTML replacement

Reply:
`td` is a standard `table` element for table data. The ticket wasn't meant to refactor the fields.njk file. Rather I just corrected the displayed line height  to be valid line height value and updated the number based the Sketch file and guidance from the designers.

Reply:
`<element class="sds-field">` is not proper HTML.

Reply:
Please review the updates as the ticket was not a typography demo refactor. Please refer to the PR summary for a full list of the branch changes.

Reply:
Correct, I am not calling for a refactor, just quality and best practices.

Reply:
I'll close this, please insure that a backlog ticket/entry is created to address this